akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -765,18 +765,26 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
         envelopePool, giveUpSendAfter, createFlightRecorderEventSink()))(Keep.right)
   }
 
-  def outboundControl(outboundContext: OutboundContext, compression: OutboundCompressions): Sink[OutboundEnvelope, (OutboundControlIngress, Future[Done])] = {
+  /**
+   * The outbound stream is defined as two parts to be able to add test stage in-between.
+   * System messages must not be dropped before the SystemMessageDelivery stage.
```

Comment:
yes, it was detected by one of the cluster tests, a Watch was dropped by the blackhole. It's alright to drop it but it must first arrive to the redelivery stage